.travis.yml

```diff
@@ -35,7 +34,6 @@ deploy:
   password:
     secure: BYpjjgOk4RZohLStyqIlP2UYCOgG76zykvxfNp2wP6ewb5Sf7HPn4BDMst1FCCPBm6ekLlIRAdZJFJp/+IPyakkWeeT/FPM6oqZxeFsa6o0aFdxzHU4qA4aEb03MUpj+7lcQ9WS8x88yyLEjxQuAN31eHNAhVWUfl59ek7Jg2zJ+m/xUmKBrVjPTkrO/PNzJ3AkcKieB/27XM6CVBfi0a+JVBrZUM+nT6WtVAwp9porcOXRj6mma7vzviyo1pIhqQkruHWDeS+iZ890KuKGwRv0QWWz5w04GVC8Yg0bTc/7kY48wHjOGiSajahdM3HjhQ5ekLwP/iUdU6ICTjfj4fHt4EJhv2TOd/P9A39Sp6Q0kUQkhp8/gyz/xBt0DdMx6XXMrUsUkP/tv7aCyRvRmm6mXrpOfODEbecbXTLG6dCdha4RluWfp5cq3Q2iYclskv0o2bwXaOvr6a7fAZO2mlKsZ/EaCJPGhKRbLe03OYo9RWvd9tPgeSrw2kYmOyc3jGuSRIBBC6PFBggK8wBnznkdYq22sI7j8xCqk8Fi/sQ7Hg1PyCIJAWQQgRUJEcMmY8eW30H7dObypORILxijUm50VoPTFcweCbM4kRTSC4+rwZOzTtH4dLGser8i4vg09JjO2iXNyDfFCaeZctTDX+pjTnEqX5wlz6zYhZUgJtwg=
   distributions: "sdist bdist_wheel"
-  skip_cleanup: true
```

Comment:
Everything else in this PR is just some code hygiene. 

This is the line that should fix #39. Travis's [skip_cleanup](https://docs.travis-ci.com/user/deployment/#uploading-files-and-skip_cleanup) option:

>When deploying files to a provider, prevent Travis CI from resetting your working directory and deleting all changes made during the build ( git stash --all) by adding skip_cleanup to your .travis.yml:

Means that if Travis rewrote the shebangs, they wouldn't get reverted back. This will ensure that the deployed code matches our repository's code. This was originally added in 6133d53c782ae2577275e4575026fbc8dee52881 because we were using a markdown README. Now that we use RST, this is not needed.